docs/source/usage/plugins/particleMergerProbalistic.rst

```diff
@@ -0,0 +1,70 @@
+.. _usage-plugins-particleMergerProbalistic:
+
+Particle Merger Probalistic version
+---------------
+
+Merges macro particles that are close in phase space to reduce computational load.
+Voronoi-based probalistic variative algorithm. The difference between Base Voronoi algorothm
+and probalistic Version in parameters: instead of threshold of spread in position and momentum
+use ratio of deleted particles. 
+
+
+.param file
+^^^^^^^^^^^
+
+In :ref:`particleMerging.param <usage-params-plugins>` is currently one compile-time parameter:
+
+===================== ====================================================================================
+Compile-Time Option   Description
+===================== ====================================================================================
+``MAX_VORONOI_CELLS`` Maximum number of active Voronoi cells per supercell.
+                      If the number of active Voronoi cells reaches this limit merging events are dropped.
+===================== ====================================================================================
+
+.cfg file
+^^^^^^^^^
+
+============================================ ================================================================================================================
+PIConGPU command line option                 Description
+============================================ ================================================================================================================
+``--<species>_merger.period``                The ouput periodicity of the plugin.
+                                             A value of ``100`` would mean an output at simulation time step *0, 100, 200, ...*.
+											 
+``--<species>_merger.ratioOfDeletedParticles``  The ratio of particles to delete. The parameter have to be in Range *[0:1]*
+
+``--<species>_merger.minParticlesToMerge``   minimal number of macroparticles can be merged into a single macroparticle.
```

Comment:
in the file `TBG_macros.cfg` this option is called `minMacroParticlesToDivide`

Reply:
This question is still open?

Reply:
@KseniaBastrakova This question is still open. IMO the documention is wrong and `minParticlesToMerge` must be `minMacroParticlesToDivide`